src/features/comments.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const initComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const addComment = createAsyncThunk(
+  'comments/addComment',
+  async (comment: Omit<Comment, 'id'>) => {
+    const value = await createComment(comment);
+
+    return value;
+  },
```

Comment:
Consider adding error handling for the `createComment` API call to manage potential failures gracefully. This will help in providing feedback to the user if the comment creation fails.